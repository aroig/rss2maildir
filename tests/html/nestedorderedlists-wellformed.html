<ol>
    <li>When the clock strikes one
        <ol>
            <li>Weird things happen</li>
            <li>Really weird things</li>
        </ol>
    </li>
    <li>And then we go on
        <ol>
            <li>first item</li>
            <li>second item</li>
        </ol>
    </li>
</dl>
