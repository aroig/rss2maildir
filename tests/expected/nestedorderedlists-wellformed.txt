  1. When the clock strikes one
      1. Weird things happen
      2. Really weird things
  2. And then we go on
      1. first item
      2. second item
